packages/api-v4/src/placement-groups/types.ts

```diff
@@ -6,32 +6,42 @@ export const AFFINITY_TYPES = {
 } as const;
 
 export type AffinityType = keyof typeof AFFINITY_TYPES;
+export type AffinityEnforcement = 'Strict' | 'Flexible';
 
 export interface PlacementGroup {
   id: number;
   label: string;
   region: Region['id'];
   affinity_type: AffinityType;
   is_compliant: boolean;
-  linode_ids: number[];
+  linodes: {
+    linode: number;
+    is_compliant: boolean;
+  }[];
+  is_strict: boolean;
 }
 
-// The `strict` parameter specifies whether placement groups should be ignored when looking for a host.
-// TODO VM_Placement: figure out the values for each create flow (create, clone, migrate etc)
-export type CreatePlacementGroupPayload = Pick<
+type PlacementGroupPayload = Pick<
```

Comment:
Can we export this type directly?

```suggestion
export type LinodePlacementGroup = Pick<
```

Reply:
sure thing, done!